readme.md

```diff
@@ -17,17 +17,18 @@ The module can be downloaded from the [releases page](https://github.com/hms-net
 * Realtime Read functionality
    * Faster tag value update rates via the Talk2M API
    * Supports up to once per second update rates
-   * Configured on a tag by tag basis
+   * Configured on a tag by tag basis by setting a tag's `Realtime` property to true
+   * Configured on a per Ewon basis by setting the `AllRealtime` tag in an Ewon's `_config` directory to true.
 * Write functionality
    * Allows Ignition to modify tag values on a Flexy
 
 ## Installation
 
-The module can be installed by navigating to the Ignition web server.  Once on the webserver go to the "Config" section then under the "System" category click the "Modules" page.  At the bottom of the "Modules" page click "Install or Upgrade a Module...".  Follow the prompts to install the module.
+The module can be installed by navigating to the Ignition web server.  Once on the webserver go to the `Config` section then under the `System` category click the `Modules` page.  At the bottom of the `Modules` page click `Install or Upgrade a Module...`.  Follow the prompts to install the module.
 
 ## Configuration
 
-Once the module is installed navigate back to the "Config" page, then under the "Tags" category click the "Ewon Connector" page.  On this page you will need to configure the connector.
+Once the module is installed navigate back to the `Config` page, then under the `Tags` category click the `Ewon Connector` page.  On this page you will need to configure the connector.
 
 ### Main
 
```

Comment:
There are still quotes around Save Changes in the Saving Changes section.

Reply:
This comment and the one below are not marked on the right lines. I couldn't comment on the correct lines since there was no diff.